test/feature/test_disk.py

```diff
@@ -1,25 +1,39 @@
+import pytest
 import torch
 
-from kornia.feature.disk import DISK
+from kornia.feature.disk import DISK, DISKFeatures
 
 
 class TestDisk:
-    def test_smoke_depth(self, device):
-        disk: DISK = DISK.from_pretrained(checkpoint='depth', device=device)
-        inp = torch.ones(1, 3, 256, 256, device=device)
+    def test_smoke(self, device):
```

Comment:
```suggestion
    def test_smoke(self, dtype, device):
```

Reply:
What would be the semantics? That we cast the input (and the network un-casts it)? Or that both inputs and network are cast and the whole computation is in `dtype`?

Reply:
Done